packages/stacks-svelte/src/components/Pagination/PaginationItem.svelte

```diff
@@ -20,17 +25,52 @@
         children?: Snippet;
     }
 
-    let { url, selected = false, onclick, children }: Props = $props();
+    let {
+        url,
+        selected = false,
+        itemNav = false,
+        onclick,
+        children,
+    }: Props = $props();
+
+    // Only used for the button variant so we're making it generic for now
+    const getClasses = (selected: boolean, itemNav: boolean) => {
+        let classes = "s-pagination--item";
+        const base = classes;
+        if (selected) {
+            classes += ` is-selected`;
+        }
+        if (itemNav) {
+            classes += ` ${base}__nav`;
+        }
+        return classes;
+    };
+
+    const classes = $derived(getClasses(selected, itemNav));
 </script>
 
 <li>
-    <a
-        class="s-pagination--item"
-        class:is-selected={selected}
-        aria-current={selected ? "page" : undefined}
-        href={url}
-        {onclick}
-    >
-        {@render children?.()}
-    </a>
+    {#if itemNav}
+        <!-- Update variant from muted to tonal when button PR goes in-->
+        <Button
+            variant="muted"
+            weight="filled"
+            class={classes}
+            aria-current={selected ? "page" : undefined}
+            href={url}
+            {onclick}
+        >
+            {@render children?.()}
+        </Button>
+    {:else}
+        <a
+            class="s-pagination--item"
+            class:is-selected={selected}
+            aria-current={selected ? "page" : undefined}
+            href={url}
+            {onclick}
+        >
+            {@render children?.()}
+        </a>
+    {/if}
```

Comment:
I don't think there's a need for the `Button` element here. We should be able to just use an anchor. We'd probably want to add `class:s-pagination--item__nav={itemNav}` to apply the nav class.